src/scss/theme.scss

```diff
@@ -33,7 +33,6 @@
 
 .navbar-light {
   background: #fff !important;
-  box-shadow: 0 0.125rem 0.25rem 0 rgba(0, 0, 0, 0.11);
```

Comment:
@ccordoba12 this line needs to be removed?

Reply:
I think so, but I'll double check.

Reply:
Maybe `!important` in the `spyder.scss` could be added in case of conflict between the rules

Reply:
Ok, will do.